### DISCUSSION TITLE
Transport connection established, but traffic only flows if I manually up the connection

### DISCUSSION THREAD

**Discussion by JMFlood** [strongswan/strongswan#2507] [General]
When setting up a host-to-host transport setup, with one seide set to "auto=start" and the other set to "auto=add", I see the connection gets established successfully.

The problem is, it seems like I need to manually run ipsec up <conn name> to get the traffic to actually being to flow. Is there a way to auto start the connection if ipsec is requested by the host without using auto=route?

**Comment by tobiasbrunner** [comment]
Please see #196.
